### PR TITLE
Feature/GitHub source code analyser

### PR DESCRIPTION
**Pull request by shivam-rawat-akto**

### CODE REVIEW

File: apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/quick_start/components/GithubSource.jsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
+import { HorizontalStack, LegacyCard, Text, TextField, VerticalStack } from "@shopify/polaris"
```
Comment (Ark2307) [inline apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/quick_start/components/GithubSource.jsx:1]: abstract out the component, it is same as bitbucket source, also copy button will give error as ref is not defined, same bitbucket.jsx

File: apps/database-abstractor/src/main/resources/struts.xml [hunk 2]
```diff
@@ -442,6 +442,17 @@
             </result>
         </action>
 
+        <action name="api/updateRepoLastRun" class="com.akto.action.CodeAnalysisAction" method="updateRepoLastRun">
+            <interceptor-ref name="json"/>
+            <interceptor-ref name="defaultStack" />
+            <result name="SUCCESS" type="json"/>
+            <result name="ERROR" type="json">
```
Comment (Ark2307) [inline apps/database-abstractor/src/main/resources/struts.xml:449]: add rbac

**Review by Ark2307 — CHANGES_REQUESTED** [review]

File: apps/database-abstractor/src/main/resources/struts.xml (hunk 2 above)
Comment (shivam-rawat-akto) [reply apps/database-abstractor/src/main/resources/struts.xml:449]: this is db-abstractor, 
we don't put rbac here

File: apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/quick_start/components/GithubSource.jsx (hunk 1 above)
Comment (shivam-rawat-akto) [reply apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/quick_start/components/GithubSource.jsx:1]: ye karte h

**Review by Ark2307 — APPROVED** [review]

**Pull request merged**